multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -321,6 +321,7 @@ def __init__(
         self._sample_data_path = sample_data_path
         self._fit_called = False  # While using ddp, after fit called, we can only use single gpu.
         self._matcher = None
+        self._hyperparameters = hyperparameters
```

Comment:
is this necessary?

Reply:
Yea, this was for saving results to model dir. It can be removed if we can get the checkpoint path. Again, is the checkpoint path self._save_path?